### ISSUE TITLE
ensure that operator bundles are published for OpenShift 4.7

### ISSUE THREAD

**Issue by errordeveloper** [cilium/cilium-olm#41]
It turns out that the bundle is currently only published for 4.5 & 4.6.

On 4.6 this works:
```
$ kubectl get  packagemanifests -n openshift-marketplace cilium 
NAME     CATALOG               AGE
cilium   Certified Operators   122m
```
But no on 4.7:
```
Error from server (NotFound): packagemanifests.packages.operators.coreos.com "cilium" not found
```

This is due to: https://github.com/cilium/cilium-olm/blob/171880aae95e95a30811a33e10f2d9c16feb9c6d/bundles/Dockerfile#L13

**Comment by errordeveloper** [comment]
Fixing this will require some minor reworking of the build automation.

**Comment by errordeveloper** [comment]
fixed in 8c216cc80120bdf148b907946df4dc44cc2c5382
